mermaid_py/client.py

```diff
@@ -209,118 +199,164 @@ def get_choices(self):
+        """
+        return self._fetch_resource("benthicattributes")
+
+    def get_fish_families(self):
+        """
+        Gets list of MERMAID fish families. Biomass constants are the calculated means of all species belonging to each
+        family.
+        :return: fish families data.
+        :rtype: dict
+        """
+        return self._fetch_resource("fishfamilies")
+
+    def get_fish_genera(self):
+        """
+        Gets list of MERMAID fish genera. Biomass constants are the calculated means of all species belonging to each
+        genus.
+        :return: fish genera data.
         :rtype: dict
         """
-        if attr not in self.attrs_endpoints:
-            raise Exception(f"Invalid attribute: {attr}")
-        return self._fetch_resource(attr)
+        return self._fetch_resource("fishgenera")
 
-    def get_projects(self, showall=False):
+    def get_fish_groupings(self):
         """
-        Gets the projects subdirectory. Without query parameters, returns a list of projects of which
+        Gets fish groupings are arbitrary (but useful) groupings of fish species, genera, and families that are treated
+        as a single taxon for purposes of observation and analysis (typically some form of “other”). As with fish genera
+        and families, biomass constants and regions are calculated from member taxa; additionally, a fish_attributes
+        property is returned listing each member species, genus, and family.
+        :return: fish groupings data.
+        :rtype: dict
+        """
+        return self._fetch_resource("fishgroupings")
+
+    def get_fish_species(self):
+        """
+        Gets list of MERMAID fish species. Includes biomass constants and maximum observed length as well as useful
+        analytical properties such as vulnerability score, trophic level, trophic group, and functional group.
+        :return: fish groupings data.
+        :rtype: dict
+        """
+        return self._fetch_resource("fishspecies")
+
+    def get_fish_sizes(self):
+        """
+        Gets Separate choice resource used only for looking up the actual size to record for a fish, given a particular
+        fish size bin used for the survey.
+        :return: fish sizes data.
+        :rtype: dict
+        """
+        return self._fetch_resource("fishsizes")
+
+    def get_projects(self, showall: bool = False):
+        """
+        Gets the projects resource. Without query parameters, returns a list of projects of which
         the user is a member. The showall query parameter may be used to return projects unfiltered by the user’s
         membership. showall is important when the user is unauthenticated.
         :param showall: (optional) Returns all projects unfiltered by the user’s membership.
         :type showall: bool
         :return: projects.
         :rtype: dict
         """
-        if not showall:
-            return self._fetch_resource("projects")
-        else:
+        if showall:
```

Comment:
Again I think this can be simpler, taking advantage of the default parameters:
```
payload = None
if showall:
    payload = "showall"
return self._fetch_resource("projects", parameters=payload)
```